### ISSUE TITLE
[1.10.2-1.1.2] Minecart Computers are labeled "entity.plethora.plethora:minecartComputer.name"

### ISSUE THREAD

**Issue by lurr** [SquidDev-CC/ccjam-2016#46]
When creating a Minecart Computer (#37), it automatically labels it as `entity.plethora.plethora:minecartComputer.name`

Image: http://i.imgur.com/Y4AdYhW.png

How to replicate:
- Place Minecart on a rail
- Put a computer inside minecart
- Open computer, type `label get`
- Label is `entity.plethora.plethora:minecartComputer.name`

Info:
Minecraft 1.10.2
Unborked 1.0.5
**Plethora 1.1.2**
CCTweaks 1.3.3

**Comment by SquidDev** [comment]
Should be fixed by ccd8842e198717190456090c815d584f15bbf0d0 (yeah, bad commit message I know). I'm not sure when this'll get released: I'm kinda wanting til CC goes open source and stabilises as I'll (hopefully) have to change things then too.

**Comment by SquidDev** [comment]
Also needs `entity.plethora:minecartComputer.name` localisation.

**Comment by lurr** [comment]
Another one to note, the default label on all CC computers, by default, is nil. This appears to cause issues saving the NBT data. On Minecarts, it is an empty string.

Manually clearing the label on a minecart appears to cause this error when saving NBT:
```[22:48:00] [Server thread/ERROR] [FML]: An Entity type org.squiddev.plethora.gameplay.minecart.EntityMinecartComputer has thrown an exception trying to write state. It will not persist. Report this to the mod author
net.minecraft.util.ReportedException: Saving entity NBT
	at net.minecraft.entity.Entity.func_189511_e(Entity.java:1728) ~[sn.class:?]
	at net.minecraft.entity.Entity.func_70039_c(Entity.java:1610) ~[sn.class:?]
	at net.minecraft.world.chunk.storage.AnvilChunkLoader.func_75820_a(AnvilChunkLoader.java:371) [avd.class:?]
	at net.minecraft.world.chunk.storage.AnvilChunkLoader.func_75816_a(AnvilChunkLoader.java:184) [avd.class:?]
	at net.minecraft.world.gen.ChunkProviderServer.func_73242_b(ChunkProviderServer.java:203) [lv.class:?]
	at net.minecraft.world.gen.ChunkProviderServer.func_186027_a(ChunkProviderServer.java:231) [lv.class:?]
	at net.minecraft.world.WorldServer.func_73044_a(WorldServer.java:1015) [lw.class:?]
	at net.minecraft.server.MinecraftServer.func_71267_a(MinecraftServer.java:366) [MinecraftServer.class:?]
	at net.minecraft.server.integrated.IntegratedServer.func_71267_a(IntegratedServer.java:216) [cch.class:?]
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:647) [MinecraftServer.class:?]
 	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:149) [cch.class:?]
 	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:482) [MinecraftServer.class:?]
 	at java.lang.Thread.run(Unknown Source) [?:1.8.0_121]
Caused by: java.lang.NullPointerException
```

When the label is nil, it also causes issues with TheOneProbe.
See [http://i.imgur.com/Qd8XCXQ.png](http://i.imgur.com/Qd8XCXQ.png)
When reloading the world, the minecart dissapears completely.

TL;DR: Localization is not fun, and when minecart labels are nil, it makes persistence impossible.

**Comment by lurquhar1221** [comment]
oh dear i just closed it
